serverless.yml

```diff
@@ -309,6 +389,7 @@ custom:
     s3BucketName: "vautomator-results"
     vautomatorQ: "vautomator-SQS"
     vautomatorDLQ: "vautomator-DLQ"
+    vautomatorSNS: "Vulnerability Assessment Results"
     ssmKms: beaede06-f6f5-4183-90bf-64a13edcea15
```

Comment:
Is this considered a secret value?

Reply:
It's not, this is just the Key ID in the KMS, so SSM uses that Key ID. Afaik there is no way to conceal it in the config, but again, there should not be any need to.